Integrations/integration-Cylance_Protect.yml

```diff
@@ -41,15 +42,24 @@ configuration:
   defaultvalue: ""
   type: 8
   required: false
+- display: Time Field
+  name: timeField
+  defaultvalue: last_found
```

Comment:
@liorkol this is the correct default value otherwise duplications will be created. if a specific customer wants to change its now possible.